pywhat/Data/regex.json

```diff
@@ -2500,6 +2471,26 @@
          "Invalid": []
       }
    },
+   {
+      "Name": "Datadog Client Token",
+      "Regex": "^(pub[a-f0-9]{32})$",
+      "plural_name": false,
+      "Description": "An [#CAE4F1][link=https://docs.datadoghq.com/logs/log_collection/javascript/]Datadog client token[/link][/#CAE4F1]",
```

Comment:
@amadejpapez @bee-san do we want to keep these descriptions?

Reply:
I think they should be removed. If a user wants to get info from these addresses, they can google the identifier name.

Reply:
Agree

Reply:
Ok